README.md

```diff
@@ -72,15 +72,17 @@ This ensures that pnpm installs dependencies in a way that is more compatible wi
 You can configure rules in your `eslint.config.js` files as in this example:
 
 ```js
-export default [
+import { defineConfig } from "eslint/config";
```

Comment:
Since eslint is still CJS, should we maybe rather use default imports (e.g., `import config from "eslint/config";`) in ESM config examples as they're guaranteed to work?

Reply:
At least locally, this seems to work fine as-is. (Node v22.9.0)